substrate/primitives/keyring/src/ed25519.rs

```diff
@@ -76,6 +83,7 @@ impl Keyring {
 		self.to_raw_public().into()
 	}
 
+	#[cfg(feature = "std")]
```

Comment:
Nit. Maybe we can simplify and remove the `sign` function from here

Reply:
I think it is fine to keep it.